Dockerfile

```diff
@@ -1,11 +1,7 @@
-FROM atddocker/atd-oracle-py:production
+FROM ghcr.io/oracle/oraclelinux8-instantclient:19
```

Comment:
nice to see we have this as an option too! 

I am wondering now which is the way to go, switching to this or using the python-oracledb package. 

https://github.com/cityofaustin/atd-data-tech/issues/12445#issuecomment-1908431197

Reply:
That's a great question @chiaberry. I am really glad you asked, because it's TIL'd me for sure. It turns out that the `python-oracledb` library is the more recent version of the `cx_oracle` python library. They renamed it. It might not be a bad idea for us do a pass over all software that connects to AMANADA and get them all up to date. This particlar task was just to get it running by updating the underlying C libraries so we could use it on ARM as well as intel, so ✅, but you've identified a great next step. Thanks for the TIL!